common/remote/sockfile.cpp

```diff
@@ -1398,7 +1398,7 @@ protected: friend class CRemoteFileIO;
             else
                 msg.append("ERROR #").append(errCode);
 #ifdef _DEBUG
-            ERRLOG("%s",msg.str());
+            OERRLOG("%s",msg.str());
```

Comment:
I think better as ERRLOG - this may be things like file not found.  @jakesmith what do you think?

Reply:
@ghalliday - Agree. I don't think this should be an operator error. ERRLOG in a #ifDEF _DEBUG statement is fine.
As a general rule, I suspect anything inside a DEBUG section should not be an operator error.